### PR TITLE
Add index.md Japanese translations

### PR DESCRIPTION
**Pull request by ystk**
This PR adds Japanese translations for index.md for the following sections. This relates to #382 . 
- Introduction
- Contributor
- Trusted Committer
- Product Owner

### CODE REVIEW

File: contributor/ja/index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: ラーニングパス - コントリビューター
+---
+コントリビュータのセクションでは、InnerSource のコントリビューターとなることの意味、優れたコントリビュータになるための心構え、コントリビューションの仕組み、およびチームや組織に対する InnerSource のメリットについて説明します。
```
Comment (johtani) [inline contributor/ja/index.md:4]:
```suggestion
コントリビューターのセクションでは、InnerSource のコントリビューターとなることの意味、優れたコントリビューターになるための心構え、コントリビューションの仕組み、およびチームや組織に対する InnerSource のメリットについて説明します。
```

長音が抜けていたので、足しました。

File: trusted-committer/ja/index.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: ラーニングパス - Trusted Committer
+---
+Trusted Committer のセクションでは、InnerSource の Trusted Committer になることの意味、およびコントリビュータをサポートする方法について説明します。
```
Comment (johtani) [inline trusted-committer/ja/index.md:4]:
```suggestion
Trusted Committer のセクションでは、InnerSource の Trusted Committer になることの意味、およびコントリビューターをサポートする方法について説明します。
```

**Review by johtani — CHANGES_REQUESTED** [review]
Left two small suggestions :)

**Comment by ystk** [comment]
Thank you very much for such a quick response, @johtani .

**Review by johtani — APPROVED** [review]
Wow, so quick ;) 
Great job, @ystk 
LGTM!

**Comment by tsadler1988** [comment]
Thanks @ystk and @johtani for doing this so quickly!

**Comment by rrrutledge** [comment]
Nice turnaround, all!

**Pull request merged**